tests/unit/test_contracts/array_tests.cpp

```diff
@@ -0,0 +1,125 @@
+#include <eosio/eosio.hpp>
```

Comment:
Please evaluate if this needs to be a toolchain test instead of a unit test (as this relates to compilation/abi gen failure)

Reply:
After look at toolchain folder, I think add new tests under toolchain better than move this test to toolchain. now using_std_array.cpp and using_std_array.json were added under toolchain/build-pass, same as what std::tuple and std::variant did.

Reply:
Is this test currently being used, or a placeholder to be used later in integration test?

Reply:
Some of theses actions are used and works well, as testin(for test array inside function), testcom( using array in multi index), testne(test nested inside function). however some of theses can't work for now as some function are not yet implemented(json to bin and bin to json, json to varient, varient to json,etc...), testepa( test array as parameter) testre(array as return) testkv(using array in kv table) as these action need data translation function so can't support for now. We will do with these in a new EPE, the test actions can't work now will be used in future.